### PR TITLE
Add some files to Craig's [original] repo

### PR DESCRIPTION
**Pull request by gretchenmanco**
Testing a pull request with three (3) files.

---

So, I added the appetizers file to my fork, and now I'm requesting of Craig that he pull my code to evaluate it for merging into his repo.  Now there are four (4) files in this my second pull request, and it appears as though there are not actually going to be TWO pull requests from Gretchen to Craig, rather ONE pull request that can be updated by Gretchen as many times as possible _prior_ to Craig actually merging or closing the pull request or Gretchen closing the pull request.  _I think_

### CODE REVIEW

**Comment by gretchenmanco** [comment]
Test comment on this pull request

File: readme.txt [hunk 1]
```diff
@@ -1 +1,44 @@
...
+## More About Markdown by G
+
+## Sub-heading
+
+### Another deeper heading
+ 
+Paragraphs are separated
+by a blank line.
+
+Two spaces at the end of a line leave a
```
Comment (cmlang42) [inline readme.txt]: I did not know that two spaces at the end of a line leave a break.

File: LOG [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+#Gretchen Manco Activities  (@gretchenmanco on TWitter)
+
+1.  Forked Craig's Repo
+2.  Starred Craig's Repo
+3.  Created a Log of Activities to Record What I broke (eek) along the way
+4.  Doing some more Markdown testing to see what happens when this code runs.  Also, trying out the various options (labels, assignees, milestones, etc.)
+5.
+6.
+7.
```
Comment (cmlang42) [inline LOG]: Maybe we should add some content on line 7. (Just asking for this to start a review.)

File: readme.txt (hunk 1 above)
Comment (gretchenmanco) [inline readme.txt]: Pretty neat, eh?

**Comment by gretchenmanco** [comment]
I made changes by incorporating the file "This is a test file" from another local techie from freeCodeCamp.  Her name is Christina!

**Comment by gretchenmanco** [comment]
C, this pullRequest is thus now to add several files, incl the one from Christina Gorton.  

Thanks so much for helping us understand.

**Comment by cmlang42** [comment]
Now that I have approved the PR (pull request). If I do nothing else it would sit at 'changes approved' but still needing "merged". You could do that if the 'team process' was setup to allow the submitter to merge their changes.

For this instance I will merge the PR as well as approve. This might mimic the role of a 'integration proxy' or 'gatekeeper' who is to do the merging.

**Pull request merged**